lib/oban/breaker.ex

```diff
@@ -45,6 +46,29 @@ defmodule Oban.Breaker do
     %{state | circuit: :enabled}
   end
 
+  @spec jitter(time :: pos_integer(), opts :: Keyword.t()) :: pos_integer()
+  def jitter(time, opts \\ []) do
+    mode = opts[:mode] || @default_jitter_mode
+    mult = opts[:mult] || @default_jitter_mult
```

Comment:
Purely for consistency with defaults elsewhere:

```suggestion
    mode = Keyword.get(opts, :mode, @default_jitter_mode)
    mult = Keyword.get(opts, :mult, @default_jitter_mult)
```